services/emailer/src/namex_emailer/email_templates/approved-ia.md

```diff
@@ -0,0 +1,69 @@
+# Results of your Name Request
+
+Your Name Request is **approved**. Follow the steps below to complete your application using this name. If the name request expires before the application is completed, a new name request will be required.
```

Comment:
Is there a check something that this NR is type "NEW" (ie, incorporation application)?

NR of types CHG and CNV will use other email templates, right?

Reply:
Yes, I updated